docs/team/brandon.md

```diff
@@ -8,10 +8,22 @@ title : Brandon - Project Portfolio Page
 The overview below follows the following standards: 
 *Roles (short for ‘in charge of role’): Description*
 
-- 
+- Development and Maintenance of Timetable Scheduling
```

Comment:
Look at my PPP to see how this is done. And look at the [recommended roles & responsibilities](https://nus-cs2113-ay2021s1.github.io/website/admin/tp-expectations.html) all the way at the bottom of the link.

Reply:
Oh I didn't see this. Thanks for pointing it out!